### ISSUE TITLE
[FEATURE]: question dock - keyboard navigation

### ISSUE THREAD

**Issue by neriousy** [anomalyco/opencode#16752]
### Feature hasn't been suggested before.

- [x] I have verified this feature I'm about to request hasn't been suggested before.

### Describe the enhancement you want to request

In desktop app you have to use your mouse in order to properly use the question dock
Users should be able to use their keyboard only if they want to
